__tests__/cli.test.js

```diff
@@ -9,19 +9,22 @@ let testId;
 let outputLogs;
 let harJSON = null;
 let metrics = null;
+let config = null;
+let goodBrowser = null;
```

Comment:
Sorry, I don't understand what `goodBrowser` and `okBrowser` mean - what is this for?